tests/io/test_read.py

```diff
@@ -262,3 +256,9 @@ def test_read_multiple_csv_with_x_only_2(self):
         assert set(adatas["dataset_num_with_missing"].obs.columns) == {"col2"}
         assert set(adatas["dataset_non_num_with_missing"].var_names) == {"strcol", "intcol", "boolcol"}
         assert set(adatas["dataset_num_with_missing"].var_names) == {"col1", "col3"}
+
```

Comment:
I'd like one or more additional tests, for example with "columns_obs_only" etc. being used